cmd/root.go

```diff
@@ -161,6 +161,35 @@ func init() {
 		"Set a request timeout for the client to connect with Kong (in seconds).")
 	viper.BindPFlag("timeout",
 		rootCmd.PersistentFlags().Lookup("timeout"))
+
+	rootCmd.PersistentFlags().String("tls-client-cert", "",
+		"Set the Kong's Admin TLS client certificate.\n"+
+			"This value can also be set using DECK_TLS_CLIENT_CERT "+
+			"environment variable. Must be used in conjunction with tls-client-key")
+	viper.BindPFlag("tls-client-cert",
+		rootCmd.PersistentFlags().Lookup("tls-client-cert"))
+
+	rootCmd.PersistentFlags().String("tls-client-cert-file", "",
+		"Path to the Kong's Admin TLS client certificate file.\n"+
```

Comment:
```suggestion
		"Path to the file containing TLS client certificate to use for authentication with Kong's Admin API.\n"+
```

Please wrap lines to ensure a pretty output.